include/packetgraph/vhost.h

```diff
@@ -66,4 +66,20 @@ const char *pg_vhost_socket_path(struct pg_brick *brick,
  */
 void pg_vhost_stop(void);
 
+/**
+ * Get RX count.
+ * This function is thread-safe.
+ * @param: brick pointer to the vhost brick
+ * @return: number of bytes received by the VM (PG -> VM)
+ */
+uint64_t pg_vhost_rx(struct pg_brick *brick);
```

Comment:
And what about the posibility to do the same with brick nix ?
I would add a brick_rx/tx in pg_brick_ops instead